AppVeyor/vsix.ps1

```diff
@@ -49,13 +62,10 @@ function Vsix-PublishToGallery{
         $repo = ""
         $issueTracker = ""
 
-        if ($env:APPVEYOR_REPO_PROVIDER -contains "github"){
+        $repoUrl = Vsix-GetRepoUrl
+        if ($baseRepoUrl -ne "") {
```

Comment:
This line looks like a typo. Shouldn't it be `$repoUrl` instead? The value `$baseRepoUrl` doesn't appear to be set anywhere in this script.